docs/src/usage.md

```diff
@@ -426,6 +426,8 @@ Each metric table has a 'hamburger' menu on the top left-hand side that can be u
 
 As *Quality-time* has to open the report in a (headless) browser and load all the metrics, creating the PDF can take some time. Especially for big reports.
 
+*Tip:* The report title in the footer of the pdf will link to the online version of the same report.
+
```

Comment:
Sphinx has mark up support for tips. It's called admonitions:

````
```{tip}
This is a tip
```
````